### PR TITLE
Update auto-user defaults to target pool scope

### PR DESCRIPTION
**Pull request by bgklein**
Switch defaults around, to align with best practices (even though default through REST is different)

### CODE REVIEW

File: src/app/components/task/base/user-identity/user-identity-picker.component.ts [hunk 1]
```diff
@@ -14,23 +14,23 @@ interface UserOption {
 
 const defaultUsers = [
     {
-        label: "Task user",
-        scope: AutoUserScope.task,
+        label: "Pool user",
```
Comment (timotheeguerin) [inline src/app/components/task/base/user-identity/user-identity-picker.component.ts]: are you doing it this way to change the order?

Comment (bgklein) [reply src/app/components/task/base/user-identity/user-identity-picker.component.ts]: Yea figured default should be first? Plus then its alphabetical too which is always a plus :)

Comment (timotheeguerin) [reply src/app/components/task/base/user-identity/user-identity-picker.component.ts]: Yeah make sense, so was the default already pool user? I don't see that changed.

Comment (bgklein) [reply src/app/components/task/base/user-identity/user-identity-picker.component.ts]: The default was 0 in this list, so just changing order and default.

Comment (timotheeguerin) [reply src/app/components/task/base/user-identity/user-identity-picker.component.ts]: sounds good, just fix the test and all good

**Pull request merged**